package.json

```diff
@@ -62,6 +62,7 @@
         "@pisa-research/server": "file:packages/server",
         "@pisa-research/test-utils": "file:packages/test-utils",
         "@pisa-research/utils": "file:packages/utils",
+        "cross-fetch": "^3.0.4",
```

Comment:
Why is this added to the root package.json?

Reply:
Oops, not sure.
Removed.